release_checklist.sh

```diff
@@ -0,0 +1,242 @@
+      issue_comment="$issue_comment"$'\n\n'"$checklist_message"
+    fi
+
+    if [[ -n "$debug_template" ]]; then
+      echo "$issue_body"
+      exit 0
+    fi
+
+    if [[ -z "$auto_confirm" ]]; then
+      echo ""
+      confirm_to_proceed "$confirm_message"$'\n\n'
+    fi
+
+    gh issue comment "$issue_number" --body "$issue_comment">/dev/null
+    gh issue edit "$issue_number" --body "$issue_body"
+  popd >/dev/null
+
+  exit 0;
+fi
+
+
+
+if [[ -z "$release_type" ]]; then
+  default_release_type="scheduled"
+
+  read -r -p "Please enter release type: scheduled|beta|hotfix [$default_release_type]:" release_type
+  release_type=${release_type:-$default_release_type}
+  echo ""
+fi
+
+if [[ ! "$release_type" == "scheduled" && ! "$release_type" == "beta" && ! "$release_type" == "hotfix" ]]; then
+    abort "Error release type must be scheduled|beta|hotfix, you entered '$release_type'"
+fi
+
+if [[ -z "$version_number" ]]; then
+  pushd "$gb_mobile_path" >/dev/null
+
+  # Ask for new version number
+  current_version_number=$(jq '.version' package.json --raw-output)
+  #version_array=($(awk -F. '{$1=$1} 1' <<< "${current_version_number}"))
+  IFS='.' read -ar version_array <<< "$current_version_number"
+  default_version_number="${version_array[0]}.$((version_array[1] + 1)).${version_array[2]}"
+
+  read -r -p "Enter the new version number [$default_version_number]: " version_number
+  echo ""
+  version_number=${version_number:-$default_version_number}
+
+  if [[ -z "$version_number" ]]; then
+      abort "Version number cannot be empty."
+  fi
+
+  popd >/dev/null
+fi
+
+# Propmt for release date if scheduled release
+if [[ -z "$release_date" && "$release_type" == "scheduled" ]]; then
+    if command -v dateround &> /dev/null; then
+      default_release_date=$(dateround today Thurs)
+
+      read -r -p "Enter the release date (YYYY-MM-DD) [$default_release_date]: " release_date
+      release_date=${release_date:-$default_release_date}
+    else
+      read -r -p "Enter the release date (YYYY-MM-DD): " release_date
+    fi
+    echo ""
+fi
+
+checklist_template_path="$script_path/templates/release_checklist.md"
+
+pushd "$gb_mobile_path" >/dev/null
+  milestone_url=$(gh api  --method GET repos/:owner/:repo/milestones --jq ".[0].html_url")
+  default_milestone_url="https://wordpress-mobile/gutenber-mobile/milestones"
+
+  milestone_url=${milestone_url:-$default_milestone_url}
+popd >/dev/null
+
+checklist_template=$(sed -e "s/{{version_number}}/${version_number}/g" -e "s/{{release_date}}/${release_date}/g" -e "s/{{milestone_url}}/${milestone_url//\//\\/}/g" "$checklist_template_path")
+
+if [[ $release_type == "beta" || $release_type == "hotfix" ]]; then
+  release_checklist_template=$(sed "/<!-- scheduled_release_only -->/,/<!-- \/scheduled_release_only -->/d" <<< "$checklist_template")
+else
+  release_checklist_template=$(sed "/<!-- non_scheduled_release_only-->/,/<!-- \/non_scheduled_release_only -->/d" <<< "$checklist_template")
```

Comment:
I noticed that `non_scheduled_release_only` parts were included in scheduled releases:

```suggestion
  release_checklist_template=$(sed "/<!-- non_scheduled_release_only -->/,/<!-- \/non_scheduled_release_only -->/d" <<< "$checklist_template")
```